src/store/state/state-model.ts

```diff
@@ -57,10 +57,10 @@ export const stateModel: StateModelIF = {
       isValidCompletingParty: true,
       isValidTransactionalFolioNumber: true,
       isValidDetailComment: true,
+      isValidSpecialResolutionConfirm: true,
       isValidCertify: false, // initially un-certified
       isValidCourtOrder: true,
-      isValidStaffPayment: true,
-      isValidSpecialResolutionConfirm: true
+      isValidStaffPayment: true
```

Comment:
This is the real fix. Order is important here!!! See line 54.

Reply:
I agree that this should fix it but I think the initial values of true are what is messing it up.

Reply:
Initial value of True for which component(s)?

My theory here is that components that aren't rendered should remain valid (True). What is your idea?

Reply:
Initial value of true for court order, it is allowed to be filed as invalid the first time but if something else like certify is caught first then it on subsequent attempts it gets caught. Is there a way we could conditionally render them as true only if they are not needed?

Reply:
I like the design where components that aren't rendered are "valid".

It sounds like Court Order isn't setting itself to False when it is rendered and it is invalid. (Same as Staff Payment.)

Reply:
I think that is part of the design... Court order only sets it's self to false after file and pay has been clicked to prevent the red styling from showing up right away.

Reply:
What is the bug with Court Order exactly?

Reply:
If you go on a review and confirm page with the CourtOrder component, and check "This filing is pursuant to a Plan of Arrangement" but don't give a court order number. Then the component is invalid but you can still complete the filing.

However if you try and file with another incomplete section like certify then the filing gets blocked, Then if you complete the certify component but not the court order component it gets caught and doesn't allow the filing to be completed.

Reply:
It sounds to me like something isn't "reacting" correctly.

Reply:
I'll create a ticket for the Court Order issue.